### ISSUE TITLE
Question : Avoid managing both my personal repository and the one in samples

### ISSUE THREAD

**Issue by spplante** [SharePoint/sp-dev-fx-webparts#281]
#### Category
- [x] Question
- [ ] Bug
- [ ] Enhancement

I am not sure if this has been covered before, but let's say I have a personal repository that I copied to the samples repository using the following contribution guidance :

https://github.com/SharePoint/sp-dev-fx-webparts/blob/master/.github/CONTRIBUTING.md#merging-your-existing-github-projects-with-this-repository

At the end, I end up having the project on my repository and the one in the samples repository. Obviously the one in the samples repository will get much more visibility, which means it will most likely have more contributions, which also means that at some point, my personal repository will end up being out of date pretty quick.

That being said, I was about to delete all the code in my personal repository, leaving only the readme file that points to the official project which is now in the samples repository, so that way I don't have two projects to maintain. That's fine for me, except the fact that I don't have my own **_Issues_** list and everything, and I find it a bit odd to be mixed with the issues list of 50+ other projects...

Is there anything I can do or I pretty much have to deal with the common issues list? I was thinking to maybe edit the readme file of the samples repository and add a "Issues" section that points the users to my personal repository (which would be empty except for a single readme file) for filling out issues so I can track them more effectively

Thanks!

**Comment by spplante** [comment]
Closing this issue, I removed the sources from my personnal repository and leaved only the readme.md file which now points the official repository in samples.

**Comment by spplante** [comment]
Actually, maybe I didn't explain this correctly, but I wasn't talking about my forked repository, like you said the only reason I created a fork was to create a pull request, I am aware that most users will see it's a fork, that's not the issue.

Originally, before creating my fork, I had started a brand new personnal repository for my project. When it was completed, I decided to add it in the SPFX samples, so following the guidance procedure, I forked the SPFX repository, executed some commands to copy my personnal repo into my forked repo, and then I did a pull request using that same forked repo.

So at the end of the day, my original personal repo (that has been copied my my forked repo), which has an issue list and everything, becomes kinda useless because the official project that will get updated by the community is the one in the samples :)

That's why I decided deleting the source code in my original personal repo, leaving only a read me file pointing to the official project in the samples repo.
